internal/util/util.go

```diff
@@ -477,6 +477,12 @@ func CheckIfVolumeSnapshotRestoresAreComplete(ctx context.Context, volumesnapsho
 				if err != nil {
 					return false, errors.Wrapf(err, fmt.Sprintf("failed to get volumesnapshotrestore %s/%s", volumesnapshotrestore.Namespace, volumesnapshotrestore.Name))
 				}
+				// check for a failed VSR
+				for _, cond := range tmpVSR.Status.Conditions {
+					if cond.Status == metav1.ConditionFalse && cond.Reason == ReconciledReasonError && cond.Type == ConditionReconciled {
```

Comment:
Are we marking the VSR status as failed ? Do we want to re-frame this log ?

Reply:
The VSR status does have a "Failed" phase now

Reply:
Awesome, if thats the case then we should directly check for the failed status in the `if` block I believe:
`if tmpVSR.Status == "failed"` instead of the condition chaining we have right now in the PR. WDYT ?

Reply:
Will do once https://github.com/migtools/volume-snapshot-mover/pull/147 is merged